cloudflare_test.go

```diff
@@ -534,3 +534,8 @@ func TestCheckResultInfo(t *testing.T) {
 		})
 	}
 }
+
+// testHTTPMethod makes sure that a request's method is what is expected.
+func testHTTPMethod(t *testing.T, expected string, r *http.Request) {
+	assert.Equal(t, expected, r.Method, "Expected method '%s', got %s", expected, r.Method)
+}
```

Comment:
Lint fails because of this. I only added this to intelligence tests, which all use GET.

Reply:
that's expected; the check here is saying that all uses of it are using the same value so it shouldn't be a parameter but instead hardcoded in. 

personally, i'm not a fan of methods like these as they take away the explictness of the tests. i'd rather for now that we continue the patterns we already have where this is asserted in the test itself.